tests/authentication_tests/test_session_cache.py

```diff
@@ -63,7 +150,10 @@ def test_url_cleanup(self):
         ) as fh:
             # Let's read the file directly to see if the data was cleaned up.
             data = yaml.load(fh)
-            self.assertEqual(data, {"current_host": "https://host.cleaned.up.on.write"})
+            self.assertEqual(
```

Comment:
There's there is also a recent host list now in the file, and the test is only interested into the current host, so only test that now.